modules/apps/portal/portal-verify-test-util/src/main/java/com/liferay/portal/verify/test/util/BaseCompanySettingsVerifyProcessTestCase.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.portal.verify.test;
+package com.liferay.portal.verify.test.util;
```

Comment:
@dantewang @yuhai 
Is there any reason we have to move this class from portal-test-integration to a separate module?

Tina